File: src/pgn.Data/Exceptions/PgnFormatException.cs
using System;

namespace ilf.pgn.Exceptions
{
    public class PgnFormatException : FormatException
    {
        public PgnFormatException() { }
        public PgnFormatException(string message) : base(message) { }
        public PgnFormatException(string message, Exception innerException) : base(message, innerException) { }
    }
}

File: src/pgn.Data/PgnFormatException.fs
namespace ilf.pgn.Exception

type PgnFormatException =
    inherit System.FormatException
    new() = { inherit System.FormatException() }
    new(message: string) = { inherit System.FormatException(message) }
    new(message: string, innerException: exn) = { inherit System.FormatException(message, innerException) }
